.vscode/launch.json

```diff
@@ -0,0 +1,212 @@
+{
```

Comment:
Please remove all configs under `.vscode/` and add a global gitignore to your environment.
So next time it won't be added no matter what gitignore file is in each individual project.

Reply:
This [link](https://docs.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) might help you.